app/controllers/users_controller.rb

```diff
@@ -1,3 +1,22 @@
 class UsersController < ApplicationController
-  
+  before_action :set_user, only: [:show]
+
+  def show
+    @dances = list_dances.sort_by { |d| d.title }
+    @dance_comments = list_dcomments.sort_by { |d| d.created_at }.reverse
+  end
```

Comment:
You may be able to replace the ruby sorting with ORDER BY at the database level:

```
list_dances.order(:title)
list_dcomments.order(created_at: :asc) # or :desc
```

https://apidock.com/rails/ActiveRecord/QueryMethods/order